browser_history/cli.py

```diff
@@ -104,34 +135,51 @@ def cli(args):
         sys.exit(1)
 
     if args.browser == "all":
-        outputs = fetch_map[args.type]()
-    else:
-        try:
-            # gets browser class by name (string).
-            selected_browser = args.browser
-            if selected_browser == "default":
-                default = utils.default_browser()
-                if default is None:
-                    sys.exit(1)
-                else:
-                    selected_browser = default.__name__
-            else:
-                for browser in utils.get_browsers():
-                    if browser.__name__.lower() == args.browser.lower():
-                        selected_browser = browser.__name__
-                        break
-            browser_class = getattr(browsers, selected_browser)
-        except AttributeError:
+        # profiles are supported only for one browser at a time
+        if args.profile is not None:
             utils.logger.error(
-                "Browser %s is unavailable." " Check --help for available browsers",
-                args.browser,
+                "Cannot use --profile option without specifying a browser"
+                " or with --browser set to 'all'"
```

Comment:
Maybe you can use `parser.error` here instead?

Reply:
Cool, `parser.error` even prints the usage which I think is very useful.